diesel.toml

```diff
@@ -3,4 +3,5 @@
 
 [print_schema]
 file = "src/schema.rs"
-import_types = ["diesel::sql_types::*", "crate::db::enums::*"]
\ No newline at end of file
+import_types = ["diesel::sql_types::*", "crate::db::enums::*"]
+filter = { except_tables = ["spatial_ref_sys"] }
```

Comment:
Let's only include the tables we are interested using `only_tables` instead of `except_tables`